laml.js

```diff
@@ -123,6 +123,13 @@ for (let name of names) {
   renamedNode.classList.add('name');
 }
 
+// convert blames to spans
```

Comment:
Just to check this is intentional: this will move the blame-span out of the heading, causing a break between heading and blame. I'm guessing you want to change the the theorem headings later?

Reply:
Ah, all I know is it looked good.